### DISCUSSION TITLE
Are bulk actions possible?

### DISCUSSION THREAD

**Discussion by K4LCIFER** [FreshRSS/FreshRSS#6230] [Q&A]
I have a lot of favorites that I would like to unfavorite. Do I need to click each one individually, or is it possible to select multiple to unfavorite all at the same time?

**Comment by Frenzie** [comment]
Something like that would make sense to add to the "mark as read" dropdown at the very least.

**Comment by math-GH** [comment]
Currently you need to click on each favorite article.

It is a very edgy case, so I do not see any chance to implement it into the core. 

Maybe an extension could solve it, but it needs to be developed first.

Another quick solution could be to run an SQL query in your database, if possible

**Reply by dphelan** [reply]
> Another quick solution could be to run an SQL query in your database, if possible

I had this use case today, and thought I'd share the solution in case anyone else needed it:

To confirm the number of affected favorites:
```sh
sqlite3 db.sqlite 'SELECT COUNT(*) FROM entry WHERE is_favorite = 1;'
```

To change their status in bulk:
```sh
sqlite3 db.sqlite 'UPDATE entry SET is_favorite = 0 WHERE is_favorite = 1;'
```

Don't forget to shut down the app first!
